td/python-scripting/pandas-df/scripts/examples.py

```diff
@@ -0,0 +1,40 @@
+# Because we do not allow custom image currently, this is how you can add 3rd party
+# libraries instead
+import sys, os
+sys.path.append('/home/td-user/.local/lib/python3.6/site-packages')
```

Comment:
[nits] Using `sys.version_info` would be more robust for future version changes.

```suggestion
py_ver = f'{sys.version_info.major}.{sys.version_info.minor}'
sys.path.append(f'/home/td-user/.local/lib/python{py_ver}/site-packages')
```